projects/step-core/src/lib/modules/dynamic-forms/pipes/dynamic-field-is-expression-allowed.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DynamicFieldType } from '../shared/dynamic-field-type';
+
+const DENY_EXPRESSION_EDITOR_FOR = [
```

Comment:
this naming sounds un-natural to me, mostly because you use verbs in it, and it looks bad also in the code when you chain it with other words. I'd call it "DENIED_FIELD_TYPES"